usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -167,15 +95,32 @@ LEFT JOIN (
     recipient_lookup AS rlv
 ) recipient_lookup ON (recipient_lookup.duns = vw_award_search.recipient_unique_id AND vw_award_search.recipient_unique_id IS NOT NULL)
 LEFT JOIN (
-    select
+    SELECT
         faba.award_id,
-        coalesce(sum(faba.transaction_obligated_amount), 0) as total_covid_obligation,
-        coalesce(sum(ffy.prior_fys_outlay), 0) + coalesce(sum(cfy.current_fy_outlay), 0) as total_covid_outlay,
-        ARRAY_AGG(faba.disaster_emergency_fund_code) AS disaster_emergency_fund_codes
-        from eligible_file_c_records faba
-        left join fy_final_outlay_balances_by_dim ffy on ffy.award_id = faba.award_id
-        left join current_fy_outlay_balance_by_dim cfy on cfy.award_id = faba.award_id
-        group by faba.award_id
+        ARRAY_AGG(DISTINCT disaster_emergency_fund_code) AS disaster_emergency_fund_codes,
+        COALESCE(sum(CASE WHEN latest_closed_period_per_fy.is_quarter IS NOT NULL THEN faba.gross_outlay_amount_by_award_cpe END), 0) AS gross_outlay_amount_by_award_cpe,
+        COALESCE(sum(faba.transaction_obligated_amount), 0) AS transaction_obligated_amount
+    FROM
+        financial_accounts_by_awards faba
+    INNER JOIN disaster_emergency_fund_code defc
+        ON defc.code = faba.disaster_emergency_fund_code
+        AND defc.group_name = 'covid_19'
+    INNER JOIN submission_attributes sa
+        ON faba.submission_id = sa.submission_id
+    INNER JOIN (
+        SELECT   submission_fiscal_year, is_quarter, max(submission_fiscal_month) AS submission_fiscal_month
+        FROM     dabs_submission_window_schedule
+        WHERE    submission_reveal_date < now() AND period_start_date >= '2020-04-01'
+        GROUP BY submission_fiscal_year, is_quarter
+    ) AS latest_closed_period_per_fy
+        ON latest_closed_period_per_fy.submission_fiscal_year = sa.reporting_fiscal_year
+        AND latest_closed_period_per_fy.submission_fiscal_month = sa.reporting_fiscal_period
+        AND latest_closed_period_per_fy.is_quarter = sa.quarter_format_flag
+GROUP BY
```

Comment:
```suggestion
        AND latest_closed_period_per_fy.is_quarter = sa.quarter_format_flag
WHERE faba.award_id IS NOT NULL
GROUP BY
```

Can limit rows being summed-across in this sub-query even further by dropping-out those that you could never join to because their `award_id` is `NULL`